cms/config.js

```diff
@@ -222,6 +223,12 @@ export default {
           required: false,
           hint: 'The actual value, this may be complex for opening hours.',
         },
+        {
+          label: 'What do other young people say?',
+          name: 'quote',
```

Comment:
Strictly speaking this is a "quotation" (n.) 

But I'm starting to think I'm the only person left in the world who cares about this.